app/controllers/pages_controller.rb

```diff
@@ -5,4 +5,38 @@ class PagesController < ApplicationController
   def show
   end
 
+  def new
+  end
+
+  def create
+    if @page.save
+      redirect_to @page
+    else
+      render :new, status: 422
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @page.update page_params
+      redirect_to @page, notice: t('page_update.notice')
+    else
+      render :edit, status: 422
+    end
+  end
+
+  def destroy
+    @page.destroy
+    redirect_to :clips, notice: t('page_delete.notice')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.